src/client/components/layout/AppHeader.tsx

```diff
@@ -40,7 +40,7 @@ const AppHeader:FunctionComponent = () => {
     { link: '/faculty', text: 'Faculty', isVisible: isReadOnly },
     { link: '/four-year-plan', text: '4 Year Plan', isVisible: true },
     { link: '/schedule', text: 'Schedule', isVisible: true },
-    { link: '/room-schedule', text: 'Room Schedule', isVisible: true },
+    { link: '/room-schedule', text: 'Room Schedule', isVisible: false },
```

Comment:
We only need to hide the tab for the public-facing site, so this should actually be:

```suggestion
    { link: '/room-schedule', text: 'Room Schedule', isVisible: isReadOnly },
```

Reply:
Additionally, this leaves the page accessible to anyone with the link, so we also need to update the `AppRouter` Component and remove the `/room-schedule` `<Route>` (line 51) and add it to the paths array in the `UnauthorizedPage` `<Route>`. 

I think we might also need to tweak the `<Route>` component on line 94 of that page to conditionally show a `ForbiddenPage`, as the other routes do.

Reply:
Good point, fixed those in [abe63e8](https://github.com/seas-computing/course-planner/pull/613/commits/abe63e81d9324fd151f60469e9dd0f2c75381918)